index.html

```diff
@@ -5,7 +5,7 @@
 
 <div class="mp-section">
   <div>
-    <h1 class="title">We enable public-purpose software and policy that is open and collaborative.</h1>
+    <h1 class="title">We support collective development and maintenance of public code in order to reduce risk, increase quality, reduce cost and increase sustainability.</h1>
```

Comment:
```suggestion
    <h1 class="title">We help public organizations collectively develop and maintain public code.</h1>
```

Reply:
Suggesting this even simpler change as the second part (benefits) are described in line 10.

Reply:
Very clear especially for a non native English speaker. Like it!!

Reply:
I like this version, but I think part of the reason we changed it to begin with is that it does not explain what public code is, which is what "public-purpose software and policy that is open and collaborative" was meant to do.  Do we think we could, without using the language that seems to make people uncomfortable, define public code?

Reply:
@nebster that appears in a separate line below this "We define ‘public code’ as open source software developed by public organizations, together with the policy and guidance needed for reuse."